### ISSUE TITLE
Logging in to phpmyadmin from my site

### ISSUE THREAD

**Issue by mvoisard** [phpmyadmin/phpmyadmin#15424]
For my website, https://maxstechandmathsite.azurewebsites.net, I've already installed phpmyadmin by cloning the repository of files from the downloaded .zip folder. Problem is, I can't log in to the MySQL server after trying to set up the credentials on the config.inc.php file which was renamed from config.inc.sample.php. Here's what the file looks like:

$cfg['Servers'][$i]['auth_type'] = 'cookie';
/* Server parameters */
$cfg['Servers'][$i]['host'] = '23.100.82.11';
$cfg['Servers'][$i]['compress'] = false;
$cfg['Servers'][$i]['AllowNoPassword'] = false;
$cfg['Servers'][$i]['user'] = 'root';
$cfg['Servers'][$i]['password'] = 'MY_PASSWORD';
$cfg['Servers'][$i]['verbose'] = 'Max\'\s Tech and Math Site';

I've tried setting the AllowNoPassword to true, but it still doesn't work. I also switched the host from localhost to the IP address I found for my server when doing an nslookup command - perhaps that is what's wrong? Am I missing some type of reference for MySQL server?

**Comment by lem9** [comment]
What's the error message you get when trying to log in?

**Comment by mvoisard** [comment]
@lem9 Actually, I figured out my problem was that I didn't even have a MySQL server instance; the IP address I was referencing - my website's - didn't have a MySQL server by default on port 3306, which I should have known. So, I set up a MySQL server from https://www.freemysqlhosting.net and copied the respective host name, username, and password into the config.inc.php file, and now I can log in just fine. Thanks for trying to help, though!
